src/counters.c

```diff
@@ -234,9 +234,15 @@ static void StatsInitCtx(void)
             SCLogDebug("Stats module has been disabled");
             SCReturn;
         }
-        const char *interval = ConfNodeLookupChildValue(stats, "interval");
-        if (interval != NULL)
-            stats_tts = (uint32_t) atoi(interval);
+        
+        intmax_t interval;
+        if (ConfGetChildValueInt(stats, "interval", &interval) == 0) {
+            stats_enabled = FALSE;
+            SCLogDebug("Stats module has been disabled, invalid interval");
```

Comment:
This should probably be an error instead? Will be hard understand for normal users otherwise.